invenio_files_rest/views.py

```diff
@@ -529,6 +549,11 @@ def create_object(self, bucket, key):
             obj = ObjectVersion.create(bucket, key)
             obj.set_contents(
                 stream, size=content_length, size_limit=size_limit)
+            # Check add tags
```

Comment:
It does not seems there's any checking of permission to create tags.

Reply:
Can we discuss this? I am not sure what you have in mind.